Makefile

```diff
@@ -69,7 +69,7 @@ else
  ifdef CONFIG_OSX
   NATIVE_TARGET = $(ARCH)-osx
   ifneq ($(CC_NAME),tcc)
-    LDFLAGS += -flat_namespace -undefined warning
+    LDFLAGS += -undefined dynamic_lookup
```

Comment:
This fixed the build using the Makefile. I can't say with confidence it's the best way. I consulted with ChatGPT.

I don't understand yet the problem with the macos meson build. I'll look again later.

Will the Makefiles eventually be removed after #4 is completed?